index.html

```diff
@@ -71,9 +91,9 @@ <h2>Web上で公開されている応募用紙リスト</h2>
         <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.3/umd/popper.min.js" integrity="sha384-ZMP7rVo3mIykV+2+9J3UJ46jBk0WLaUAdn689aCwoqbBJiSnjAK/l8WvCWPIPm49" crossorigin="anonymous"></script>
         <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.1.1/js/bootstrap.min.js" integrity="sha384-smHYKdLADwkXOn1EmN1qk/HfnUcbVRZyYmZ4qpPea6sjB/pTJ0euyQp0Mk8ck+5T" crossorigin="anonymous"></script>
         <script>
-        function generateApplicationList(year){
-            $.getJSON(`data/${year}/application.json` , function(data) {
-                var ulObj = $(`#application${year}`);
+        function generateApplicationList(year, kind = ''){
```

Comment:
It may be a good idea to change the default value of `kind` to something like "zenkoku" or "general" to align the directory structure of "next".

Reply:
Thanks for reviewing.
I'm planning to rewrite this project entirely to use Static Site Generator such as Gatsby, so this function will be removed soon.

Reply:
I see, that's a great idea! Thanks for working on it!
Then it completely looks good to me.